include/networkit/graph/Graph.hpp

```diff
@@ -473,6 +473,8 @@ class Graph final {
 
         NeighborIterator(std::vector<node>::const_iterator nodesIter) : nIter(nodesIter) {}
 
+        NeighborIterator() {}
```

Comment:
Please add a comment that this c'stor is required for Python (and why) and should not be used directly.

Reply:
Please make that a doxygen comment.